apps/src/sites/studio/pages/home/courses.js

```diff
@@ -0,0 +1,26 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Courses from '@cdo/apps/templates/teacherHomepage/Courses';
+import {initCourseExplorer} from '@cdo/apps/courseExplorer/courseExplorer';
+
+$(document).ready(showCourses);
+
+function showCourses() {
+  // Initialize the non-React Course/Tool Explorer component code.
+  initCourseExplorer();
+
+  const coursesData = document.querySelector('script[data-courses]');
+  const configCourses = JSON.parse(coursesData.dataset.courses);
+
+  const englishTeacherData = document.querySelector('script[data-englishteacher]');
```

Comment:
This implies that you're getting these two sets of data (courses/englishteacher) from two separate scripts, which you are not. I would either (a) use a script with one data attribute that has both courses and englishteacher in it or (b) get rid of line 16 and reuse coursesData. I would also rename these better.